### DISCUSSION TITLE
Filter out drivers in live timing

### DISCUSSION THREAD

**Discussion by Turrican76** [f1multiviewer/issue-tracker#19] [Feature requests] [answered]
I think it would be a really nice addition if there was a way to select drivers to be shown in the build-in Live Timing. This way you could select the drivers you really care about, as Live Timing can be pretty overwhelming with all the data and it doesn`t quite fit in a smaller window with all 20 drivers shown. Maybe some kind of checkbox selection like in the Driver Cam selection.

**Comment by LapsTimeOFF** [comment, marked as answer]
Hey, this was discretely implemented a few versions ago, in the live timing window, click on the drivers you want to keep to turn them blue, then press `F`
